.github/workflows/build.yml

```diff
@@ -423,3 +423,49 @@ jobs:
         run: |
           set -eu -o pipefail
           go test -v ./integration/multiarch --addr=docker:// --copa="$(pwd)/copa" -timeout 0
+
+  test-patch-podman:
```

Comment:
should we add this as a matrix in existing workflows instead of standalone workflow?